modules/olm-deprecations-schema.adoc

```diff
@@ -7,6 +7,8 @@
 
 The optional `olm.deprecations` schema defines deprecation information for packages, bundles, and channels in a catalog. Operator authors can use this schema to provide relevant messages about their Operators, such as support status and recommended upgrade paths, to users running those Operators from a catalog.
 
+When this schema is defined, the {product-title} web console displays warning badges for the affected elements of the Operator, including any custom deprecation messages, on both the pre- and post-installation pages of the OperatorHub.
```

Comment:
When this schema is defined, the web console displays warning notifications, including any custom deprecation messages, for deprecated Operator Packages, Channels, and Versions on both the pre- and post-installation pages of the OperatorHub. The deprecated badge is displayed next to the product logo and {product-title}, and the warning icons are displayed next to the Channels and Versions dropdown, as well as on the Subscription detail update channel link and modal. The custom deprecation messages for Operator Packages, Channels, and Versions are displayed in the warning alert.
cc @jhadvig

Reply:
I dont think we need to specify the location of the deprecation elements(badges, icons, ...).
I think having a general awareness of the feature is sufficient.

Reply:
@adellape Never mind the suggested change.

Reply:
No worries, I wrestled with how specific to get here as well. Thanks!